packages/core/src/actors.ts

```diff
@@ -504,25 +391,98 @@ export function createBehaviorFrom<TEvent extends EventObject>(
 ): Behavior<TEvent, undefined>;
 export function createBehaviorFrom(entity: Spawnable): Behavior<any, any> {
   if (isPromiseLike(entity)) {
-    return createPromiseBehavior(() => entity);
+    return fromPromise(() => entity);
   }
 
   if (isObservable<any>(entity)) {
-    return createObservableBehavior(() => entity);
+    return fromObservable(() => entity);
   }
 
   if (isStateMachine(entity)) {
-    return createMachineBehavior(entity);
+    return fromMachine(entity);
   }
 
   if (isFunction(entity)) {
-    return createCallbackBehavior(entity);
+    return fromCallback(entity);
   }
 
   throw new Error(`Unable to create behavior from entity`);
 }
 
-interface SpawnBehaviorOptions {
+interface CreateActorRefOptions {
   id?: string;
   parent?: ActorRef<any>;
 }
+
+export function createActorRef<TEvent extends EventObject, TEmitted>(
```

Comment:
q: what makes `createActorRef` special - I mean, why is this only used by `useSpawn`? Is there a potential on unifying this with something?

Reply:
I'm thinking we should actually remove `useSpawn` in the v5-compatible versions of `@xstate/react` and `@xstate/vue` - it doesn't make sense to have them.